src/java.base/share/classes/java/text/NumberFormat.java

```diff
@@ -929,14 +929,18 @@ public int getMaximumIntegerDigits() {
 
     /**
      * Sets the maximum number of digits allowed in the integer portion of a
-     * number. maximumIntegerDigits must be &ge; minimumIntegerDigits.  If the
-     * new value for maximumIntegerDigits is less than the current value
-     * of minimumIntegerDigits, then minimumIntegerDigits will also be set to
-     * the new value.
+     * number during formatting. {@code maximumIntegerDigits} must be &ge;
+     * {@code minimumIntegerDigits}. If the new value for {@code
+     * maximumIntegerDigits} is less than the current value of
+     * {@code minimumIntegerDigits}, then {@code minimumIntegerDigits} will
+     * also be set to the new value.
+     *
+     * @apiNote
```

Comment:
I think this is `@implNote` as this is for subclass implemntors, not the users of this API. (apply to other locations)